content/SCALE/SCALETutorials/Shares/SMB/SMBPrivateDatasetShare.md

```diff
@@ -0,0 +1,211 @@
+
+TrueNAS does not control what Windows allows through the File Explorer.
+If the personal directories show in File Explorer, use [Windows file properties](https://community.spiceworks.com/t/how-to-configure-file-sharing-in-windows-server/1014185) and access control to hide the folder in the share.
+
+After creating a dataset and SMB share, use the path to this dataset as the home directory for each user with a private share directory.
+Other users who do not require a private personal share can use a path to another dataset for their home directory.
+This replaces the legacy home share personal home directory when connecting to the share.
+
+Home directories are not visible to or accessible by other users.
+Private directory created for users do not show in the datasets table on the **Datasets** screen.
+hey are visible on the **Edit User** screen in the file explorer section under the **Home Directory** field.
+
+Other options for configuring individual user directories include:
+* Configure a single share on the TrueNAS and provision individual user directories on the client OS. 
+* Create a single SMB share and configure the ACL so that users can create individual directories on the share that inherit write access for the user and grant read access to the administrator.
+* Create an SMB share using the **Private SMB datasets and shares** preset, and then create per-user datasets under the umbrella of a single share when users access the share.
+
+Creating an SMB private dataset and share requires provisioning users or joining Active Directory, and configuring the system storage and share.
+
+## Adding Private SMB Datasets and Shares
+
+Private directories are not intended for every user on the system.
+When using the **Private SMB Datasets and Shares** share **Purpose** preset and configuring it as a home directory the system TrueNAS does not show the private directories to all users with access to the root level of the share.
+
+Examples of setting up private SMB shares are those for backups, system configuration, and users or departments that need to keep information private from other users.
+
+You can manually add users and groups in TrueNAS, or configure groups in Active Directory and add users to each group, and then have AD add the users and group to TrueNAS.
+After adding users and groups, configure private directories and home directories for the users and group(s).
+Before setting up SMB shares, check system alerts to verify no errors related to connections to Active Directory are listed.
+Resolve any issues with Active Directory before proceeding. If Active Directory cannot bind with TrueNAS you cannot start the SMB service after making changes.
+
+To add private shares and datasets for users that require home directories:
+
+1. Create the share (and dataset) using the **Private SMB Datasets and Shares** preset.
+
+2. Configure the share dataset ACL to use the **NFSv4_HOME** preset and add the user as an ACL entry to the ACL.
```

Comment:
Here, it says "add the user as an ACL entry" for the dataset used by a "private smb share".
I think it could be clarified *which user* should be added to the ACL.

I'm pretty sure you don't need to add one there, except if you want a group (e.g. builtin_administrators) to always have access to the underlying datasets as well (which will have the owner set to the SMB user accessing the private share).